docs/managing_deis/upgrading-deis.rst

```diff
@@ -53,6 +53,13 @@ Finally, update ``deisctl`` to the new version and reinstall:
     In-place upgrades incur approximately 10-30 minutes of downtime for deployed applications, the router mesh
     and the platform control plane.  Please plan your maintenance windows accordingly.
 
+.. note::
+
+    On Amazon EC2, Deis now enables the :ref:`PROXY protocol <proxy_protocol>` by default. If
+    upgrading a cluster older than Deis v1.4, you can enable this by running the command
+    ``deisctl config router set proxyProtocol=1`` and configuring the ELB to allow traffic
+    at ``TCP 443:443``.
```

Comment:
You also need to enable the proxy protocol policy for the ELB and change the targets from HTTP to TCP. I think it would be easier to create new ELB using the cloudformation template and link it to the existing auto-scaling group

Changes:
https://github.com/deis/deis/commit/4f634ed1275c47c60ec5070124c681923525b78a